roles/confluent.common/tasks/debian.yml

```diff
@@ -10,6 +10,9 @@
   shell: apt-get install apt-transport-https -y --force-yes
```

Comment:
Should we use the APT module here?  It supports forcing installs.

Reply:
will try

Reply:
So I just tried and the apt module works without the force flag with our debian image...
I don't want to make a change thats too destructive but I think we are fine to just replace this task with 
```
- name: Install apt-transport-https
  apt:
    name: apt-transport-https
  tags: package
```

The reason I had it as the shell module was a previous debian image (in vagrant) i used did not work with the apt module.

Wondering your thoughts?

Reply:
Yeah I think we should change it to the APT module, it's not that big of a change IMO.

Reply:
ok done